eventmesh-webhook/eventmesh-webhook-receive/src/main/java/org/apache/eventmesh/webhook/receive/WebHookController.java

```diff
@@ -93,7 +93,7 @@ public void execute(String path, Map<String, String> header, byte[] body) throws
 
         if (!Objects.equals(webHookConfig.getContentType(), header.get(CONTENT_TYPE))) {
             throw new Exception(
-                "http request header content-type value is mismatch. current value " + header.get(CONTENT_TYPE));
+                    "http request header content-type value is mismatch. current value " + header.get(CONTENT_TYPE));
         }
```

Comment:
I wonder why this place needs one more TAB? It seems the original version already has one. 

Also, the spotless format rule **conflicts with** IDEA's. IDEA will re-format the spotless formated code to the original version:

![image](https://github.com/apache/eventmesh/assets/41445332/2aab570f-f979-4c1c-928c-51e0999e05eb)

Many similar changes are made like this.

Reply:
On the second question, I think the spotless conversion rules belong to the **constraints of eventmesh** for developers' coding specifications, and do not necessarily have to be exactly the same as IDEA's own rules. 
We can use the IDEA plug-in Eclipse Code Formatter to replace the original formatting function of IntelliJ IDEA with format rule, so that in the process of writing code can be formatted at any time, which greatly improves development efficiency.

Reply:
Alright. I wonder why another TAB is needed in this place then?

Reply:
Because the current alignment length format strategy is 16.
Due to a newline between `throw new Exception (` and `http request xxx`, the alignment length is 16 (Based on the position of the first letter of `throw`), but in fact, there should be no newline at the beginning of this string when coding.

Similar behavior can be found in references such as **WebHookController.class, line 115**, `WebhookProtocolTransportObject `and `.cloudEventId`, because it is exactly 16 alignment lengths, no conversion is triggered.

Reply:
The bug of indentation has been fixed in the latest commit.